### PR TITLE
4.7

### PR DESCRIPTION
**Pull request by sophattack**

### CODE REVIEW

File: src/tactic/ui/app/search_limit_wdg.py [hunk 1]
```diff
@@ -281,7 +281,8 @@ def get_display(self):
         #widget.add_style("border", "solid 1px blue")
         widget.add_color("background", "background")
         widget.add_color("color", "color")
-        widget.add_style("padding: 10px")
+        widget.add_style("padding: 5px 8px 13px 30px")
+        widget.add_style("position", "relative")
```
Comment (CeltonMcGrath) [inline src/tactic/ui/app/search_limit_wdg.py]: delete this

File: src/tactic/ui/app/search_limit_wdg.py [hunk 2]
```diff
@@ -545,6 +546,7 @@ def get_display(self):
         self.set_as_panel(top)
         top.set_unique_id()
         top.add_class("spt_search_limit_top")
+        top.add_style("position: relative")
```
Comment (CeltonMcGrath) [inline src/tactic/ui/app/search_limit_wdg.py]: verify if necessary, delete if not

File: src/tactic/ui/app/search_limit_wdg.py [hunk 3]
```diff
@@ -712,6 +714,7 @@ def get_display(self):
 
 
         top.add_class("spt_table_search")
+        top.add_style("position: relative")
```
Comment (CeltonMcGrath) [inline src/tactic/ui/app/search_limit_wdg.py]: verify or delete

File: src/tactic/ui/container/popup_wdg.py [hunk 4]
```diff
@@ -1020,7 +1020,7 @@ def get_onload_js(self):
 
     // get the content container
     var width_wdg = popup.getElement(".spt_popup_width");
-    width_wdg.setStyle("min-width", "200px");
+    width_wdg.setStyle("min-width", "900px");
```
Comment (CeltonMcGrath) [inline src/tactic/ui/container/popup_wdg.py]: The better way to use this is to use the width argument.

File: src/tactic/ui/container/tab_wdg.py [hunk 5]
```diff
@@ -1300,6 +1300,7 @@ def get_styles(self):
 
             .spt_tab_header.drag-header {
                 position: relative;
+                height:35px;
```
Comment (CeltonMcGrath) [inline src/tactic/ui/container/tab_wdg.py]: verify and delete if not necessary

File: src/tactic/ui/panel/collection_wdg.py [hunk 6]
```diff
@@ -1043,7 +1041,18 @@ def get_display(self):
         collection = Search.get_by_search_key(self.collection_key)
 
         top = self.top
-        top.add_style("min-height: 400px")
+        top.add_style("height", "530px")
```
Comment (CeltonMcGrath) [inline src/tactic/ui/panel/collection_wdg.py]: try to find a way to not hardcode the height ( please :) )

File: src/tactic/ui/panel/table_layout_wdg.py [hunk 7]
```diff
@@ -954,10 +954,13 @@ def get_display(self):
...
                 scroll.add_attr("spt_window_resize_offset", window_resize_offset)
 
+            window_resize_xoffset = self.kwargs.get("window_resize_xoffset")
+            if window_resize_xoffset:
+                scroll.add_attr("spt_window_resize_xoffset", window_resize_xoffset)
+
             # sync header to this scroll
             # FIXME: this does not work with locked columns as the locked columns have their own
             # scrollbar
-            scroll.add_class("spt_table_scroll")
```
Comment (CeltonMcGrath) [inline src/tactic/ui/panel/table_layout_wdg.py:960]: Why is this removed?

File: src/tactic/ui/tools/sobject_wdg.py [hunk 8]
```diff
@@ -811,15 +811,15 @@ def get_config_xml(self):
...
                   <display class='tactic.ui.panel.CustomLayoutWdg'>
                   <html>
-                    <div style="padding: 20px">
+                    <div style="padding: 20px;">
                     <div style="font-size: 25px">Notes</div>
                     <div>List of all of the notes for this item</div>
                     <hr/>
                     <br/>
-                    <div style="min-height: 200px">
+                    <div style="overflow:auto; height:500px;" spt_window_resize_offset="300" class="spt_window_resize spt_notes_wrap">
```
Comment (CeltonMcGrath) [inline src/tactic/ui/tools/sobject_wdg.py]: remove this... we willl have to create our own notes wdg;

File: src/tactic/ui/widget/discussion_wdg.py [hunk 9]
```diff
@@ -875,7 +875,7 @@ def get_styles(self):
 
             .spt_notes_dialog {
                 border: solid 1px #DDD;
-                width: 100%;
```
Comment (CeltonMcGrath) [inline src/tactic/ui/widget/discussion_wdg.py]: maybe remove for now since we are going to have to create a custom notes wdg

File: src/tactic/ui/panel/table_layout_wdg.py (hunk 7 above)
Comment (sophattack) [reply src/tactic/ui/panel/table_layout_wdg.py:960]: because the same class is added twice
